spec/rubocop/cop/rspec/example_wording_spec.rb

```diff
@@ -329,6 +329,8 @@
       DESC
       end
     RUBY
+
+    expect_no_corrections
```

Comment:
Is this what we want, though? Does it discourage people from implementing autocorrect, or will it work fine as a “placeholder” until autocorrect is implemented.

Reply:
This method only useful for cops that implement autocorrect but not for all cases. If someone doesn't implement autocorrect to begin with, there will also be no offense. There's no cop to check for `extend AutoCorrector` (;

In this specific case, I think autocorrect could theoretically be implemented but hasn't because heredocs are hard and nobody actually writes tests with heredoc text descriptions.